fe/fe-core/src/main/java/org/apache/doris/analysis/TableRef.java

```diff
@@ -79,55 +79,48 @@
 public class TableRef implements ParseNode, Writable {
     private static final Logger LOG = LogManager.getLogger(TableRef.class);
     protected TableName name;
-    private PartitionNames partitionNames = null;
-
     // Legal aliases of this table ref. Contains the explicit alias as its sole element if
     // there is one. Otherwise, contains the two implicit aliases. Implicit aliases are set
     // in the c'tor of the corresponding resolved table ref (subclasses of TableRef) during
     // analysis. By convention, for table refs with multiple implicit aliases, aliases_[0]
     // contains the fully-qualified implicit alias to ensure that aliases_[0] always
     // uniquely identifies this table ref regardless of whether it has an explicit alias.
     protected String[] aliases;
-
+    protected List<Long> sampleTabletIds = Lists.newArrayList();
     // Indicates whether this table ref is given an explicit alias,
     protected boolean hasExplicitAlias;
-
     protected JoinOperator joinOp;
     protected List<String> usingColNames;
-    private ArrayList<String> joinHints;
-    private ArrayList<String> sortHints;
-    private ArrayList<String> commonHints; //The Hints is set by user
```

Comment:
Why moved those codes?

Reply:
Just moved the position, after the code was formatted automatically

Reply:
![image](https://user-images.githubusercontent.com/13197424/192817703-713a25f2-5491-41d4-a7d7-06342a6ac2e8.png)